<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QWidget" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>777</width>
    <height>653</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>注册</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/icon.qrc">
    <normaloff>:/newPrefix/签到.jpeg</normaloff>:/newPrefix/签到.jpeg</iconset>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>751</width>
     <height>371</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLabel" name="ShowLb">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>581</width>
      <height>341</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="icon/icon.qrc">:/newPrefix/tips.jpg</pixmap>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>751</width>
     <height>241</height>
    </rect>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>150</y>
      <width>431</width>
      <height>71</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="StartBt">
       <property name="text">
        <string>新建</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="PhotoBt">
       <property name="text">
        <string>拍照</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="RegisterBt">
       <property name="text">
        <string>注册</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="ModelBt">
       <property name="text">
        <string>模型训练</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>211</width>
      <height>118</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
          </font>
         </property>
         <property name="text">
          <string>学号：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>16</pointsize>
           <weight>50</weight>
           <bold>false</bold>
          </font>
         </property>
         <property name="text">
          <string>姓名：</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="stu_id">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
           <strikeout>false</strikeout>
          </font>
         </property>
         <property name="statusTip">
          <string comment="请输入学号"/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="stu_name">
         <property name="font">
          <font>
           <pointsize>14</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>100</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="MsgLb">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="statusTip">
     <string extracomment="请输入学号"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="icon/icon.qrc"/>
 </resources>
 <connections/>
</ui>
